src/utils/pr.test.ts

```diff
@@ -5,13 +5,13 @@ describe("pr", () => {
   it("should check for the correct checkbox value", () => {
     const prDescription = `
 This PR was opened by the [ready-release-go](https://github.com/woodpecker-ci/plugin-ready-release-go) plugin.
-When you're ready to do a release, you can merge this and a release and tag with version
-will be created automatically.If you're not ready to do a release yet, that's fine,
-whenever you add more changes to this PR will be updated.
+When you're ready to do a release, you can merge this and a new release with version '1.2.3'
+will be created automatically. If you're not ready to do a release yet, that's fine,
```

Comment:
The "that's fine" still sounds a bit strange to me as it sounds a bit like talking to a child 😅 But no big deal overall...

Reply:
Funny, yeah now as you say that. I just copied it from the changeset bot, but guess we leave it for now.